docs/references/javascript/organization/organization.mdx

```diff
@@ -16,8 +16,10 @@ This is a class that represents an organization. It's returned from the [`create
 | `slug` | `string \| null` | The organization slug. If supplied, it must be unique for the instance. |
 | `logoUrl` | `string \| null` | The full URL for the organization logo. (deprecated in favor of `imageUrl`) |
 | `imageUrl` | `string` | Holds the organization logo or default logo. |
+| `hasImage` | `boolean` | A getter boolean to check if the organization has a uploaded image. Returns `false` if Clerk is displaying an avatar for the organization. |
```

Comment:
```suggestion
| `hasImage` | `boolean` | A getter boolean to check if the user has uploaded an image or one was copied from OAuth. Returns `false` if Clerk is displaying an avatar for the user. |
```

Reply:
This is not the user -- this is the organization. The 'an' is a correct edit, but this hasImage refers to the organization's image and not the user.

Reply:
oh good point, I was sifting through too quickly - thank you for catching that!